backend/aethel_db/views.py

```diff
@@ -20,10 +20,17 @@ def aethel_status():
     return dataset is not None
 
 
+@dataclass
+class AethelSamplePhrase:
+    index: str
```

Comment:
shouldn't this be an int?

Reply:
because it's serialized in a list, you could use the list index instead of an explicit field. but there's an argument to be made for clarity

Reply:
You are right that this information is duplicated since all phrases are passed anyway (so you don't need it to unique identify phrases). I prefer to remove it, unless you think it is better to keep it.